apps/frontend/src/routes/login/index.tsx

```diff
@@ -97,3 +97,48 @@ export default component$(() => {
     </div>
   );
 });
+
+export const head: DocumentHead = {
+  title: 'The FREE Open-Source URL Shortener | Reduced.to',
```

Comment:
I think that the title name should be something like "Reduced.to | Login".
Also please make sure to set up relevant description as well.